server/www/templates/posts/preview.html

```diff
@@ -18,7 +18,7 @@
         </div>
         <div class="post-text">
             <a href="/posts/{{post.id}}">
-                <h3>{{post.title}}</h3>
+                <h3>{{ post.title || post.content | limitTo:50 }}{{ post.content.length > 50 ? ' ...' : ''}}</h3>
```

Comment:
@middle8media post title can be up to 150 chars. `post.content.length > 50 ? ' ...' : ''` will only show an ellipsis if we're using content. Could you do `post.content.length > 50 || post.title.length > 50 ? ' ...' : ''` instead?